### PR TITLE
Add background for stage 1

### PR DESCRIPTION
**Pull request by mark0978**
Trying to help people understand what stage 1 means when picking presets.

This is a documentation change only.  Licensed under MIT License.
[skip ci]

### CODE REVIEW

**Comment by xtuc** [comment]
Thanks for your PRs.

Just an idea, I think we would be better to move this in [Babel's FAQ](http://babeljs.io/docs/faq/) and add a link to the corresponding section in stage-1,2,3 presets. What do you think?

Related PRs: #5252 #5253

**Comment by mark0978** [comment]
I added it to those 3 pages because that where you end up when searching for babel stage 1.  Putting it on the faq doesn't help that.

File: packages/babel-preset-stage-1/README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 > Babel preset for stage 1 plugins.
 
+You can read more about the stages @ http://www.2ality.com/2015/11/tc39-process.html but the gist of stage 1 is:
```
Comment (hzoo) [inline packages/babel-preset-stage-1/README.md]: These get rendered on http://babeljs.io/docs/plugins/preset-stage-1/#install so we'd want to use the markdown link syntax for these.

**Comment by hzoo** [comment]
Thanks for this - we also have info already at http://babeljs.io/docs/plugins/#presets-stage-x-experimental-presets-

**Comment by hzoo** [comment]
@rauschma the information here is basically just copied from your article so don't know how you wanted this to be added? Maybe we should change the wording to say something about how it's taken from your article?

**Comment by existentialism** [comment]
Yea, +1 on adding attribution

**Comment by mark0978** [comment]
That was why I added the link back to the article in question.  Because it is just a tiny portion of the information on that page.

**Comment by existentialism** [comment]
Sure, but I think we're suggesting maybe something more than than just saying "here".

Perhaps something like:

> As detailed by @rauschma's excellent [overview of the TC39 process](http://www.2ality.com/2015/11/tc39-process.html), the gist of Stage 1 is:

?

**Comment by rauschma** [comment]
It’s a difficult balance: you want to at least mention the name of the person you are quoting, but you also don’t want to clutter the content. One solution is to attribute more extensively, but at the end:

---

Further reading and source of this page:

* Chapter “[The TC39 process for ECMAScript features](http://exploringjs.com/es2016-es2017/ch_tc39-process.html)” in “Exploring ES2016 and ES2017” by Axel Rauschmayer
* (Etc.)
* (Etc.)

**Comment by hzoo** [comment]
Ok that sounds good to me

We kinda do this reference with the proposals https://github.com/babel/babel/blob/master/packages/babel-plugin-transform-do-expressions/README.md#references

**Comment by mark0978** [comment]
That seems good to me.

**Comment by hzoo** [comment]
Made an update

**Pull request merged**